tests/test_edgeql_expressions.py

```diff
@@ -3085,6 +3085,32 @@ class TestExpressions(tb.QueryTestCase):
                 r"SELECT 'bb\   aa';"
             )
 
+    @test.xfail('The string literal is missing some content')
+    async def test_edgeql_expr_string_12(self):
+        # Issue #1269
+        await self.assert_query_result(
+            r'''SELECT 'bb\
+aa \
+            bb';
+            ''',
+            ['bbaa bb'],
+        )
+
+    @test.xfail('alternative newline is not accepted after line continuation')
+    async def test_edgeql_expr_string_13(self):
+        # Issue #1269
+        #
+        # \r or \n should both be allowed after line continuation symbol
+        await self.assert_query_result(
+            "SELECT 'bb\\\n   aa';",
+            ['bbaa'],
+        )
+
+        await self.assert_query_result(
+            "SELECT 'bb\\\r   aa';",
```

Comment:
This should probably has `\r\n` or another test for `\r\n` because turning `'a\\\r\nb` into `a\nb` sounds bad.